modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/Invites/index.js

```diff
@@ -39,6 +52,32 @@ const Invites = () => {
 		? steps.dxpCloud
 		: steps.successDxpCloud;
 
+	const accountRoles = useMemo(() => {
+		let filterRoles = [
+			...new Set(
+				rolesData?.accountAccountRoles?.items.map(({name}) => name)
+			),
+		];
+		const SLA_CURRENT = project.slaCurrent;
+		const isPartner = project.partner;
+
+		if (
+			!SLA_CURRENT.includes('Gold') &&
```

Comment:
I saw many times the use of Gold, Platinum in this file, this could be a constant

Reply:
Never mind, I saw you created in another commit

Great :D